src/rgw/rgw_reshard.cc

```diff
@@ -834,8 +857,24 @@ int RGWReshard::process_single_logshard(int logshard_num)
     ldout(store->ctx(), 5) << __func__ << "(): failed to acquire lock on " << logshard_oid << dendl;
     return ret;
   }
-
-  utime_t lock_start_time = ceph_clock_now();
+  Clock::time_point lock_start_time = Clock::now();
+  const std::string& log_func_name = __func__;
```

Comment:
I hate myself slightly, but shouldn't this be passed as const char*?

Reply:
Yes, because that's what GCC provides it as, isn't it....

Reply:
could use string_ref perhaps